### PR TITLE
Responsive minor update

### PR DESCRIPTION
**Pull request by Tulun**
Improvements to overall responsiveness of voter-dapp.

https://app.clubhouse.io/uma-project/story/1632/fix-the-module-for-committing-votes-on-mobile

Screen Shot 2021-07-14 at 1 26 48 PM
Screen Shot 2021-07-14 at 1 26 53 PM

### CODE REVIEW

**Review by daywiss — COMMENTED** [review]
damn that was a lot of css, looks really great though in action. one small thing, im not sure if this was intentional but the break might need to be slightly sooner: 
image @Tommy1231232

File: src/common/components/modal/Modal.tsx [hunk 1]
```diff
@@ -9,23 +9,25 @@ type Props = {
...
-          <ExitButton onClick={onClose}>
-            <Times />
-          </ExitButton>
-          <Content>{children}</Content>
-        </Wrapper>
-        <BgBlur coords={{ y: window.scrollY }} />
-      </Portal>
-    );
-  };
+const _Modal: ForwardRefRenderFunction<
```
Comment (Gamaranto) [inline src/common/components/modal/Modal.tsx:12]: Question unrelated to the code, are those just formatting changes? Github shows the diff but the code looks identical to me

File: src/common/components/modal/Modal.tsx [hunk 2]
```diff
@@ -49,10 +51,17 @@ export const Wrapper = styled.aside<StyledProps>`
...
+  @media screen and (max-width: 768px) {
+    margin: 2rem;
+  }
 `;
 
 const Content = styled.div`
   ${tw`pb-10 px-5`};
+  @media screen and (max-width: 768px) {
+    width: 100%;
+    /* margin: 2rem; */
```
Comment (Gamaranto) [inline src/common/components/modal/Modal.tsx:63]: Can prob remove

File: src/features/vote/styled/ActiveRequests.styled.tsx [hunk 3]
```diff
@@ -87,6 +87,11 @@ export const Table = styled.table<TableProps>`
       font-size: 14px;
       line-height: 1.57;
       opacity: 0.5;
+      @media screen and (max-width: 768px) {
+        /* border-right: 0; */
```
Comment (Gamaranto) [inline src/features/vote/styled/ActiveRequests.styled.tsx:91]: Can prob remove

File: src/features/vote/styled/RevealPhase.styled.tsx [hunk 4]
```diff
@@ -7,6 +7,8 @@ interface StyledProps {
 export const Wrapper = styled.div<StyledProps>`
   &.RequestPhase {
     padding: 1rem 4rem;
+    width: 100%;
+    overflow-y: scroll;
```
Comment (Gamaranto) [inline src/features/vote/styled/RevealPhase.styled.tsx:11]: I think `overflow-y: auto` might be better here as `scroll` will always show a sidebar while `auto` will only show if the content overflows

**Review by Gamaranto — COMMENTED** [review]
Tons of good CSS! Just some minor comments

**Comment by Gamaranto** [comment]
Nit, I think the menu animation is slightly out of sync
CleanShot 2021-07-15 at 12 44 43

File: src/common/components/modal/Modal.tsx (hunk 1 above)
Comment (Tulun) [reply src/common/components/modal/Modal.tsx:12]: Probably from prettier.

**Comment by Tulun** [comment]
> Nit, I think the menu animation is slightly out of sync
> CleanShot 2021-07-15 at 12 44 43

Fixed :)

**Review by daywiss — APPROVED** [review]

**Review by Gamaranto — APPROVED** [review]
LGMT

**Pull request merged**